pkg/plugins/golang/domain.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package golang
+
+import "sigs.k8s.io/kubebuilder/v3/pkg/plugins"
+
+// DefaultGoNameQualifier is the suffix appended to all kubebuilder plugin names.
+const DefaultGoNameQualifier = "go" + plugins.DefaultNameQualifier
```

Comment:
`plugins.DefaultNameQualifier` is `.kubebuilder.io` and `golang.DefaultGoNameQualifier` is `go.kubebuilder.io`. We should choose if the `Default*NameQualifier` consts have the `.` or not and stick with that for all of them.